config/jobs/kubernetes/kops/templates/periodic.yaml.jinja

```diff
@@ -37,6 +37,7 @@
           -v 2 \
           --up --down \
           --cloud-provider={{cloud}} \
+          --admin-access=0.0.0.0/0 \
```

Comment:
Why?
```suggestion
          --admin-access=0.0.0.0/0 \
```

Reply:
https://prow.k8s.io/view/gs/kubernetes-jenkins/logs/e2e-ci-kubernetes-e2e-cos-gce-slow-canary/1707442458176000000

The addon test is failing because it can't ssh to the master instance. This should allow SSH from anywhere, including the IPs of the nodes which aren't known to us. I assume that test runs in a pod.

Reply:
I suspect there are few other e2e tests that assume ssh is reachable from anywhere

Reply:
can you limit this to gce for now?

Reply:
Done